src/tools/namespace_tool.cpp

```diff
@@ -118,7 +120,8 @@ int NameSpaceTool::RunCommand(const std::string &cmd) {
             return 0;
         }
     } else if (cmd == kCreateCmd) {
-        return core_->CreateFile(fileName, FLAGS_fileLength * mds::kGB);
+        return core_->CreateFile(fileName, FLAGS_fileLength * mds::kGB,
+                             FLAGS_stripeUnit, FLAGS_stripeCount);
```

Comment:
If there are requirements for these two parameters(FLAGS_stripeUnit, FLAGS_stripeCount), please do some pre-checks.

Reply:
> If there are requirements for these two parameters(FLAGS_stripeUnit, FLAGS_stripeCount), please do some pre-checks.

The two new parameters are not required（they are optional) and mds has done some check,so pre-check is no need

Reply:
> > If there are requirements for these two parameters(FLAGS_stripeUnit, FLAGS_stripeCount), please do some pre-checks.
> 
> The two new parameters are not required（they are optional) and mds has done some check,so pre-check is no need

This mds-side check can be copied here.